.env

```diff
@@ -19,7 +19,7 @@ APP_SECRET=c1cdcd0a21a1ffeee4c1dc26b5fe228b
 ###< symfony/framework-bundle ###
 
 ###> symfony/mailer ###
-# MAILER_DSN=smtp://localhost
+MAILER_DSN=smtp://contact@1gscop.fr
```

Comment:
The SMTP server should probably in a .local env because each deployment of the project will probably use it's own SMTP server